hadoop-hdfs-project/hadoop-hdfs/src/test/java/org/apache/hadoop/hdfs/server/namenode/TestDecommissioningStatus.java

```diff
@@ -453,10 +493,10 @@ public void testDecommissionLosingData() throws Exception {
     BlockManagerTestUtil.recheckDecommissionState(dm);
     // Block until the admin's monitor updates the number of tracked nodes.
     waitForDecommissionedNodes(dm.getDatanodeAdminManager(), 0);
```

Comment:
`java.util.concurrent.TimeoutException` happened in this line in my environment. Could you check it?